_posts/2018-04-04-QtWebEngine-on-Rpi3.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "QtWebEngine on the Raspberry Pi 3 in a Wayland client"
+date:   2018-04-04 19:00:00 +0100
+categories: yocto qt
```

Comment:
maybe also add: webengine, rpi, wayland

Reply:
JFYI, the URLs to the posts are based on categories and date, so this one would be pelux.io/yocto/qt/2018/04/04/<title> or even /yocto/qt/webengine/rpi/wayland.

I know, it sucks. Dunno if it is worth changing or if we should just keep the categories short.